spec.html

```diff
@@ -20014,6 +20022,8 @@ <h1>Runtime Semantics: Evaluation</h1>
         <emu-grammar>UpdateExpression : LeftHandSideExpression `++`</emu-grammar>
         <emu-alg>
           1. Let _lhs_ be ? Evaluation of |LeftHandSideExpression|.
+          1. [id="step-postfix-increment-web-compat-insertion-point", normative-optional] If the host is a web browser or otherwise supports <emu-xref href="#sec-late-errors-for-function-call-assignment-targets" title></emu-xref>, then
```

Comment:
Actually I don't think this guard is even necessary, as it should be implied by `~web-compat~`, right?

Reply:
You are quite right; I inserted this in a later commit after second-guessing myself about how explicit to be. I can definitely drop it.

Reply:
Do you still prefer that these be highlighted as normative optional?
(Or does that make it look like one could implement just part of this feature?)